api/services/account_service.py

```diff
@@ -321,7 +321,7 @@ def get_current_tenant_by_account(account: Account):
         return tenant
 
     @staticmethod
-    def switch_tenant(account: Account, tenant_id: int = None) -> None:
+    def switch_tenant(account: Account, tenant_id: Optional[int] = None) -> None:
```

Comment:
The `tenant_id` here is not allowed be None

Reply:
It's alright to leave `tenant_id` nullable here. Since it's checked later at Line 328 introduced from #2363.